README.md

```diff
@@ -1,3 +1,20 @@
+##Install TL;DR
+
+```shell
+pip install pre-commit detect-secrets
+mkdir .gds
+detect-secrets scan > .gds/.secrets.baseline
+detect-secrets audit .gds/.secrets.baseline
+pre-commit install -c .gds/.pre-commit-config.yaml #this file is from this repo
```

Comment:
Some people have been confused by where the .pre-commit-config.yaml comes from, maybe we should make this really clear and put it in a separate step ?